test/integration/kubernetes_deploy_test.rb

```diff
@@ -802,6 +802,18 @@ def test_deploy_successful_with_partial_availability
     assert_equal(1, new_ready_pods.length, "Expected exactly one new pod to be ready, saw #{new_ready_pods.length}")
   end
 
+  def test_deploy_successful_with_multiple_template_dirs
+    result = deploy_dir(fixture_path("test-partials"), fixture_path("cronjobs"),
+      bindings: { 'supports_partials' => 'yep' })
+    assert_deploy_success(result)
+  end
+
+  def test_deploy_successful_with_multiple_template_dirs_multiple_partials
+    result = deploy_dir(fixture_path("test-partials"), fixture_path("test-partials2"),
```

Comment:
The goal of this test is to ensure that partials with identical names are actually rendered according to the relative path of the given template directory. This was done by using an identically-named partial that renders different parts of the spec; if the 'wrong' partial is used, the deploy would fail. This convinces me enough that the renderer for a given template directory is behaving as expected (e.g. properly scoped to the right dir)